diff --git a/dist/client.js b/dist/client.js
index 21b3800b0352ecca965826be4e37c9cc97b21190..fc94fb1b50d6e6a6e74b50ed7e69bcbb1b35c4b6 100644
--- a/dist/client.js
+++ b/dist/client.js
@@ -26,6 +26,8 @@ export function simplifyProperty(prop) {
             return prop.date?.start ?? null;
         case "people":
             return (prop.people ?? []).map((p) => p.id);
+        case "relation":
+            return (prop.relation ?? []).map((r) => r.id);
         default:
             return null;
     }
